osu.Game/Database/RealmArchiveModelImporter.cs

```diff
@@ -338,11 +338,11 @@ public async Task<IEnumerable<Live<TModel>>> Import(ProgressNotification notific
                     // import to store
                     realm.Add(item);
 
+                    PostImport(item, realm);
```

Comment:
This was introduced in https://github.com/peppy/osu/commit/8b6665cb5b211c06e24f9d3e9bf210480bc08760